src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -408,6 +408,10 @@ $stickyHeaderHeight: 55px;
       align-items: center;
       justify-content: center;
       width: 100%;
+
+      img {
+        width: 41px;
+      }
```

Comment:
Image is in the NoMetricData component, but the style will be applied only in DetailsMetrics. The style should be in NoMetricData